npm/translatePolicy.go

```diff
@@ -856,7 +856,7 @@ func translateEgress(ns string, targetSelector metav1.LabelSelector, rules []net
 }
 
 // Drop all non-whitelisted packets.
-func getDefaultDropEntries(ns string, targetSelector metav1.LabelSelector) []*iptm.IptEntry {
+func getDefaultDropEntries(ns string, targetSelector metav1.LabelSelector, isIngress, isEgress bool) []*iptm.IptEntry {
```

Comment:
isIngress -> hasIngress
isEgress -> hasEgress
might be more appropriate.

Reply:
fixed